src/MC/fix_bond_create_angle.h

```diff
@@ -0,0 +1,39 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef FIX_CLASS
+
+FixStyle(bond/create/angle,FixBondCreateAngle)
+
+#else
+
+#ifndef LMP_FIX_BOND_CREATE_ANGLE_H
+#define LMP_FIX_BOND_CREATE_ANGLE_H
+
+#include "fix_bond_create.h"
+
+namespace LAMMPS_NS {
+
+class FixBondCreateAngle : public FixBondCreate {
+ public:
+  FixBondCreateAngle(class LAMMPS *, int, char **);
+
+ private:
+  double angle1,angle2,amin,amax;
```

Comment:
Not clear why this line is here?  Angle12 are just local to the one method in this class, so could be declared there?  And amin,amax are defined in the parent header as protected so will be visible here.  So this is duplicate declaration which is bad I believe.  Amin, amax also thus do not be passed to this class's constrain method?

Reply:
You are right. This was probably a remnant from previous code rejigs.